### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ebroder**
Cleanup a Meteor package that we were only barely using and some type declarations that got merged upstream.

### CODE REVIEW

**Comment by ebroder** [comment]
And snuck in one more quick change - a postshrinkwrap script that corrects SHA1 integrity hashes in `package-lock.json` back to SHA512.

File: tests/acceptance/lib.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* eslint-disable import/prefer-default-export */
 /* eslint-disable no-console */
+import { promisify } from 'util';
```
Comment (zarvox) [inline tests/acceptance/lib.ts:3]: I guess this comes from the meteor node stubs?  IIRC `util` is a strictly-nodejs thing.

File: tests/acceptance/profiles.tsx [hunk 2]
```diff
@@ -15,6 +16,21 @@ const addUserToRole = new TypedMethod<{ userId: string, scope: string, role: str
 const createHunt = new TypedMethod<{ name: string }, string>('test.methods.profiles.createHunt');
 const joinHunt = new TypedMethod<{ huntId: string, userId: string }, void>('test.methods.profiles.joinHunt');
 
+const subscribeAsync = (name: string, ...args: any[]) => new Promise<Meteor.SubscriptionHandle>(
+  (resolve, reject) => {
+    const handle = Meteor.subscribe(name, ...args, {
+      onError: (reason?: Meteor.Error) => {
```
Comment (zarvox) [inline tests/acceptance/profiles.tsx:22]: Per https://docs.meteor.com/api/pubsub.html#Meteor-subscribe there is no `onError`, only `onStop` (which fires with no args on client-driven teardown and a Meteor.Error if the publish function throws or the server cancels the sub).

**Review by zarvox — COMMENTED** [review]
Loving this -- super excited to be getting rid of the monkeypatches (and dropping the coffeescript dependency to boot).

It looks like the removal of the WebApp type decls might have made `imports/server/assets.ts` no longer typecheck?  Might need an explicit `any` or some other type hint on https://github.com/deathandmayhem/jolly-roger/blob/395d9baea3026f792b64352fb949d38d60882b2b/imports/server/assets.ts#L85

File: tests/acceptance/lib.ts (hunk 1 above)
Comment (ebroder) [reply tests/acceptance/lib.ts:3]: Yeah, correct on both counts.

Comment (ebroder) [reply tests/acceptance/lib.ts:3]: (Specifically, `meteor-node-stubs` pulls in https://npmjs.com/package/util)

**Comment by ebroder** [comment]
> It looks like the removal of the WebApp type decls might have made `imports/server/assets.ts` no longer typecheck? Might need an explicit `any` or some other type hint on
> 
> https://github.com/deathandmayhem/jolly-roger/blob/395d9baea3026f792b64352fb949d38d60882b2b/imports/server/assets.ts#L85

Ah yeah - I forgot that upstream requested that I declare that as `unknown` instead of `any`, so there's a bit more type nudging required. Pushing a fix momentarily.

**Review by zarvox — APPROVED** [review]
Lovely, thanks for doing this!

**Pull request merged**